Integrations/integration-Aella_StarLight.yml

```diff
@@ -0,0 +1,331 @@
+            demisto.setLastRun(cur_t)
+        else:
+            demisto.info('Poll incidents failed {}'.format(action_result))
+            demisto.incidents([])
+
+    def aella_get_event_command():
+        demisto.info('Aella started get-event with {}'.format(demisto.args()['eid']))
+        event_id = demisto.args()['eid']
+        query_json = { 'query': { 'match': { '_id': event_id } } }
+        end_point = URL + '/{0}/{1}/_search'.format('aella-ser*', 'amsg')
+
+        action_result = {}
+        make_rest_call(end_point,
+            USERNAME, PASSWORD, action_result, data=query_json
+        )
+        demisto.info('REST API query eid returns {}'.format(action_result))
+        if action_result['status'] == 'Success':
+            demisto.info('Run Query is successful')
+            response = action_result.get('data')
+            timed_out = response.get('timed_out', False)
+            hits = response.get('hits', {}).get('hits', [])
+            for item in hits:
+                index = item.get('_index', '')
+                source = item.get('_source', {})
+                if index:
+                    source['_index'] = index
+                    source['timed_out'] = timed_out
+                    demisto.debug('This is my run_query result aellaEvent {}'.format(source))
+                    demisto.results({
+                        'Type': entryTypes['note'],
+                        'ContentsFormat': formats['json'],
+                        'Contents': source,
+                        'HumanReadable': tableToMarkdown('aellaEvent', source),
+                        'EntryContext': {
+                            'Aella.Event(val["_id"]==obj["_id"])': source
+                        }
+                    })
+        else:
+            demisto.info('Run Query failed'.format(action_result))
+
+    ''' EXECUTION CODE '''
+    demisto.info('Command is {}'.format(demisto.command()))
+
+    # The command demisto.command() holds the command sent from the user.
+    if demisto.command() == 'test-module':
+        # This is the call made when pressing the integration test button.
+        action_result = {}
+        make_rest_call(URL + '/_cluster/health',
+            USERNAME, PASSWORD, action_result
+            )
+
+        if action_result['status'] == 'Success':
+            demisto.results('ok')
+        else:
+            demisto.results('failed')
+
+    if demisto.command() == 'fetch-incidents':
+        fetch_incidents_command()
+
+    if demisto.command() == 'aella-get-event':
+        aella_get_event_command()
+
+    # You can use demisto.args()[argName] to get a specific arg. args are strings.
+    # You can use demisto.params()[paramName] to get a specific params.
+    # Params are of the type given in the integration page creation.
+  type: python
+  commands:
+  - name: aella-get-event
+    arguments:
+    - name: eid
+      required: true
+      description: event id from the Star Light incident
+    outputs:
```

Comment:
There are many outputs returned to context, but only a few of them are defined here - is this on purpose?
If we don't need these outputs in context, please handle that in the code.

Reply:
My understanding that defined output is just for better user experience. I would like to keep all the fields available, but I do not want to define every fields explicitly. So I picked some fields I think might be used more frequently.